mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -1,3 +1,486 @@
+	};
+
+	class vn_b_wheeled_m274_01_03 {
+		tags[] = {"m274","transport","car","airport"};
+	};
+
+	class vn_b_wheeled_m274_02_01 {
+		tags[] = {"m274","transport","car"};
+	};
+
+	class vn_b_wheeled_m274_02_03 {
+		tags[] = {"m274","transport","car","airport"};
+	};
+
+	class vn_b_wheeled_m274_mg_01_01 {
+		tags[] = {"m274","armed","firesupport","car"};
+	};
+
+	class vn_b_wheeled_m274_mg_02_01 {
+		tags[] = {"m274","armed","firesupport","car","antitank"};
+	};
+
+	class vn_b_wheeled_m274_mg_03_01 {
+		tags[] = {"m274","armed","firesupport","car"};
+	};
+
+	class vn_b_wheeled_m54_01 {
+		tags[] = {"m54","transport","truck"};
+	};
+
+	class vn_b_wheeled_m54_02 {
+		tags[] = {"m54","transport","truck"};
+	};
+
+	class vn_b_wheeled_m54_03 {
+		tags[] = {"m109","transport","truck"};
+	};
+
+	class vn_b_wheeled_m54_01_airport {
+		tags[] = {"m54","transport","truck", "airport"};
+	};
+
+	class vn_b_wheeled_m54_ammo {
+		tags[] = {"m54","ammo","truck"};
+	};
+
+	class vn_b_wheeled_m54_ammo_airport {
+		tags[] = {"m54","ammo","truck","airport"};
+	};
+
+	class vn_b_wheeled_m54_fuel {
+		tags[] = {"m49","fuel","truck"};
+	};
+
+	class vn_b_wheeled_m54_fuel_airport {
+		tags[] = {"m49","fuel","truck","airport"};
+	};
+
+	class vn_b_wheeled_m54_mg_01 {
+		tags[] = {"m54","armed","firesupport","truck"};
+	};
+
+	class vn_b_wheeled_m54_mg_02 {
+		tags[] = {"m54","armed","firesupport","truck","antiair"};
+	};
+
+	class vn_b_wheeled_m54_mg_03 {
+		tags[] = {"m54","armed","firesupport","truck"};
+	};
+
+	class vn_b_wheeled_m54_repair {
+		tags[] = {"m54","repair","truck"};
+	};
+
+	class vn_b_wheeled_m54_repair_airport {
+		tags[] = {"m54","repair","truck","airport"};
+	};
+
+	class vn_c_bicycle_01 {
+		tags[] = {"bicycle","civilian","transport","bicycle"};
+	};
+
+	class vn_c_bicycle_02 {
+		tags[] = {"bicycle","civilian","transport","bicycle"};
+	};
+
+	class vn_c_car_01_01 {
+		tags[] = {"boheme","civilian","transport","car"};
+	};
+
+	class vn_c_car_01_02 {
+		tags[] = {"boheme","civilian","transport","car"};
+	};
+
+	class vn_c_car_02_01 {
+		tags[] = {"boheme","civilian","transport","car"};
+	};
+
+	class vn_c_car_03_01 {
+		tags[] = {"boheme","civilian","transport","car"};
+	};
+
+	class vn_c_car_04_01 {
+		tags[] = {"boheme","civilian","transport","car"};
+	};
+
+	class vn_c_wheeled_m151_01 {
+		tags[] = {"offroad","civilian","transport","car"};
+	};
+
+	class vn_c_wheeled_m151_02 {
+		tags[] = {"offroad","civilian","transport","car"};
+	};
+
+	class vn_i_armor_type63_01 {
+		tags[] = {"type63","armed","firesupport","tank","lighttank"};
+	};
+
+	class vn_i_fank_70_static_sgm_high_01 {
+		tags[] = {"sgm","static","machinegun"};
+	};
+
+	class vn_i_fank_70_static_sgm_low_01 {
+		tags[] = {"sgm","static","machinegun"};
+	};
+
+	class vn_i_fank_70_static_sgm_low_02 {
+		tags[] = {"sgm","static","machinegun"};
+	};
+
+	class vn_i_fank_70_static_zgu1_01 {
+		tags[] = {"zgu1","static","antiair","nodisassemble"};
+	};
+};
```

Comment:
This is not a complete list of BN vehicles we want to spawn in... we need to add some of the missing items from the commented out block below it